zipkin-storage/zipkin2_cassandra/src/main/java/zipkin2/storage/cassandra/Schema.java

```diff
@@ -99,6 +101,11 @@ static KeyspaceMetadata getKeyspaceMetadata(Session session) {
   }
 
   static KeyspaceMetadata ensureExists(String keyspace, boolean searchEnabled, Session session) {
+    session.getCluster().getMetadata().getAllHosts().forEach((host) -> {
+      Preconditions.checkState(
+              0 < VersionNumber.parse("3.11.3").compareTo(host.getCassandraVersion()),
+              "All Cassandra nodes must be running 3.11.3+");
+    });
```

Comment:
I was a bit confused when writing this…
Isn't this type of check done somewhere else in the codebase?